pkgs/development/libraries/wayland/default.nix

```diff
@@ -20,6 +20,14 @@ in stdenv.mkDerivation rec {
     sha256 = "0k995rn96xkplrapz5k648j651wc43kq817xk1x8280h16gsfxa6";
   };
 
+  patches = [
+    # Fix documentation to be reproducible.
+    (fetchpatch {
+      url = "https://gitlab.freedesktop.org/wayland/wayland/-/commit/f54158c10f687ceedaf07a348af362b9b25b97b4.patch";
```

Comment:
Could we use https://gitlab.freedesktop.org/wayland/wayland/-/commit/e53e0edf0f892670f3e8c5dd527b3bb22335d32d.patch from the merged commit in the master branch instead? Just to be sure that the other commit won't get garbage collected.

Other than that it LGTM, thanks!

Reply:
Ah, yes. It hadn’t been merged at the time I opened the PR. (I checked the hash, and it didn’t change, btw — didn’t just leave it)